core/assets/jsons/save.json

```diff
@@ -49,80 +49,129 @@
     "path": "levels/shooter-tutorial.json"
   },
   {
-    "name": "level 5",
+    "name": "Cave level 8",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
     "path": "levels/easyLevel.json"
   },
   {
-    "name": "level 6",
+    "name": "Cave level 9",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
     "path": "levels/mediumLevel.json"
   },
   {
-    "name": "level 7",
+    "name": "Cave level 10",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
     "path": "levels/hardLevel.json"
   },
   {
-    "name": "level 8",
+    "name": "Cave level 11",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
-    "path": "levels/enemySurroundingPlayer.json"
+    "path": "levels/enemySurroundingPlayerUpdated.json"
   },
   {
-    "name": "level 9",
+    "name": "Cave level 12",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
-    "path": "levels/forceSneak.json"
+    "path": "levels/forceSneakUpdated.json"
   },
   {
-    "name": "level 10",
+    "name": "Cave level 13",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
-    "path": "levels/shooterAndPatrol.json"
+    "path": "levels/Jalil-11.json"
   },
   {
-    "name":       "level 11",
+    "name":       "Cave level 14",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
-    "path":       "levels/maxLevel.json"
+    "path":       "levels/medium.json"
   },
   {
-    "name": "level 12",
+    "name": "Cave level 15",
     "world":      0,
     "unlocked":   true,
     "completed":  true,
-    "path": "levels/onlypatrol.json"
+    "path": "levels/Jalil-12.json"
   },
   {
-    "name": "level 13",
+    "name": "Cave level 16",
+    "world":      0,
+    "unlocked":   true,
+    "completed":  true,
+    "path": "levels/Jalil-14.json"
+  },
+  {
+    "name": "Fores level 1",
```

Comment:
Tbh the last two levels of cave were much harder than this one

Reply:
ask Jalil. May not be bad to have a slight dip in diff. Also Jalil-14 could be volcano?